### PR TITLE
add preemption case when mutli-preeptor-job with different priority c…

### PR DESCRIPTION
**Pull request by Thor-wl**
…omes

Signed-off-by: Thor <1187526662@qq.com>

### CODE REVIEW

File: test/e2e/preempt.go [hunk 1]
```diff
@@ -260,4 +262,60 @@ var _ = Describe("Job E2E Test", func() {
...
+					min: 1,
+					rep: rep,
+				},
+			},
+		}
+
+		job.name = "j1-q1"
+		job.pri = lowPriority
+		job.queue = "q1-preemption"
+		queue1Job := createJob(ctx, job)
```
Comment (hzxuzhonghu) [inline test/e2e/preempt.go:293]: I guess you mean low priority job

File: test/e2e/preempt.go [hunk 2]
```diff
@@ -260,4 +262,60 @@ var _ = Describe("Job E2E Test", func() {
...
+		err := waitTasksReady(ctx, queue1Job, int(rep))
+		Expect(err).NotTo(HaveOccurred())
+
+		job.name = "j2-q1"
+		job.pri = middlePriority
+		job.queue = "q1-preemption"
+		job.tasks[0].rep = rep / 2
+		job.tasks[0].min = rep / 2
+		queue2Job := createJob(ctx, job)
```
Comment (hzxuzhonghu) [inline test/e2e/preempt.go:302]: same here: middle priority job

File: test/e2e/preempt.go [hunk 3]
```diff
@@ -260,4 +262,60 @@ var _ = Describe("Job E2E Test", func() {
...
+		Expect(err).NotTo(HaveOccurred())
+		err = waitTasksReady(ctx, queue1Job, int(rep)/2)
+		Expect(err).NotTo(HaveOccurred())
+
+		job.name = "j3-q1"
+		job.pri = highPriority
+		job.queue = "q1-preemption"
+		job.tasks[0].rep = rep
+		job.tasks[0].min = rep
+		queue1Job3 := createJob(ctx, job)
```
Comment (hzxuzhonghu) [inline test/e2e/preempt.go:313]: ditto

File: test/e2e/preempt.go (hunk 1 above)
Comment (Thor-wl) [reply test/e2e/preempt.go]: new PR has optimize code style to make case more readable

**Pull request merged**